### DISCUSSION TITLE
[Discussion] Love the effort!

### DISCUSSION THREAD

**Discussion by Frans-L** [agencyenterprise/react-native-health#79] [General]
Hey, This isn't an issue. I just wanted to say thank you for developing this library further!  😊

**Comment by lucaspbordignon** [comment]
Hey @Frans-L ! I'm super glad that you liked it, want to help the community as much as we can. And also, feel free to give any suggestion or register any issue you might face in the repository, so we can have the project running in the long-term!

Thank you a lot for your support! 😄
